dashboards/management/commands/data_quality.py

```diff
@@ -53,10 +60,15 @@ def total_bills_per_session(bills, chamber):
         # In case bills don't have actions
         if bill_with_latest_action.actions.count() > 0:
             latest_action = bill_with_latest_action.actions.latest("date")
-            latest_action_date = datetime.datetime.strptime(
-                latest_action.date, "%Y-%m-%d"
-            )
-            latest_action_date = pytz.UTC.localize(latest_action_date)
+            # 2020-05-06
```

Comment:
in all cases it'd be safe to truncate to 10 chars and save, might be better to just make that the default (even if already 10, but also handle cases where date is another number of characters due to lacking a timezone or inclusion of milliseconds)

Reply:
Fixed!